qiskit/compiler/transpile.py

```diff
@@ -226,6 +256,25 @@ def callback_func(**kwargs):
     return circuits
 
 
+def any_delay_in(circuits: Union[List[QuantumCircuit], QuantumCircuit]) -> bool:
```

Comment:
is this still used?

Reply:
It is used in `execure()`. I'll move it to `execute.py` and change it private.

Reply:
1f20746